corePlugins/alignContent.js

```diff
@@ -1,6 +1,6 @@
 import generator from '../util/generator';
 
-export default generator.generate('content', 'alignContent', [
+export default () => generator.generate('content', 'alignContent', [
```

Comment:
Just wondering why the closure export and not the generated object 🤔

Reply:
Because of dynamic configuration. To make all core plugin align and be in same way I did this. Because of dynamic conflict some were dependent on theme or colour but some were not. Now as all are arrow functions all core plugins behaves in same way.